src/serlio/modifiers/MayaCallbacks.cpp

```diff
@@ -378,6 +382,17 @@ void updateMayaMesh(double const* const* uvs, size_t const* uvsSizes, uint32_t c
 	outputMesh.setMetadata(newMetadata);
 }
 
+void copyStringToWCharPtr(const std::wstring input, wchar_t* result, size_t& resultSize) {
+	if (resultSize <= input.size())    // also check for null-terminator
+		resultSize = input.size() + 1; // ask for space for null-terminator
+#if _MSC_VER >= 1400
+	wcsncpy_s(result, resultSize, input.c_str(), resultSize);
+#else
+	wcsncpy(result, pathStr.c_str(), resultSize);

```

Comment:
does not compile, forgot to rename `pathStr`

Reply:
fixed typo